emperor/support_files/js/controller.js

```diff
@@ -262,12 +263,40 @@ define([
       selector: '#' + scope.$divId.attr('id') + ' .emperor-plot-wrapper',
       trigger: 'none',
       items: {
+        'saveState': {
+          name: 'Save Emperor setup',
+          icon: 'edit',
+          callback: function(key, opts) {
```

Comment:
Noted, thanks!!

On (May-02-16|20:58), Joshua Shorenstein wrote:
> > @@ -262,12 +263,40 @@ define([
> >        selector: '#' + scope.$divId.attr('id') + ' .emperor-plot-wrapper',
> >        trigger: 'none',
> >        items: {
> > +        'saveState': {
> > +          name: 'Save Emperor setup',
> > +          icon: 'edit',
> > +          callback: function(key, opts) {
> 
> The scope changes if this is done, and ends up as the div instead of the function. The wrapper function counteracts that.
> 
> ---
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/emperor/pull/466/files/8c43ce45e9e5ab5b86574f7bb97dd4257117935e#r61836354